package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "warehouse.ai",
+  "name": "@mluther1/warehouse.ai",
```

Comment:
Gottem. Please remove.

Reply:
🤦‍♂️ 
![Ugh, not again](http://i.imgur.com/HLSuopc.gif)
Nice Catch.